google/cloud/aiplatform/training_jobs.py

```diff
@@ -935,54 +994,7 @@ def __init__(
         self._script_path = script_path
         self._staging_bucket = staging_bucket
 
-    def _create_input_data_config(
```

Comment:
Consolidated this function by removing it and augmenting the base class's version